### DISCUSSION TITLE
PaddleOCR/applications/%E5%BF%AB%E9%80%9F%E6%9E%84%E5%BB%BA%E5%8D%A1%E8%AF%81%E7%B1%BBOCR

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#13858] [Q&A]
# PaddleOCR/applications/%E5%BF%AB%E9%80%9F%E6%9E%84%E5%BB%BA%E5%8D%A1%E8%AF%81%E7%B1%BBOCR

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/applications/%E5%BF%AB%E9%80%9F%E6%9E%84%E5%BB%BA%E5%8D%A1%E8%AF%81%E7%B1%BBOCR.html

**Comment by awakening12585** [comment]
大佬你好,可以分享下 上面提到的 关于 head修改 和 修改loss的 改动代码吗?
